### PR TITLE
login and remember

### PR DESCRIPTION
**Pull request by hngda01**

### CODE REVIEW

**Comment by hngda01** [comment]
image

**Comment by huynhduc96** [comment]
ok

File: app/models/user.rb [hunk 1]
```diff
@@ -27,6 +29,35 @@ class User < ApplicationRecord
...
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update_attributes remember_digest: User.digest(remember_token)
+  end
+
+  def authenticated? remember_token
+    return false if remember_digest.nil?
+    BCrypt::Password.new remember_digest.is_password?(remember_token)
```
Comment (thanhhungchu95) [inline app/models/user.rb]: `BCrypt::Password.new(remember_digest).is_password? remember_token`

File: config/routes.rb [hunk 2]
```diff
@@ -2,5 +2,9 @@
   scope "(:locale)", locale: /en|vi/ do
     root "static_pages#home"
     get "static_pages/home"
+    get    "/login",   to: "sessions#new"
```
Comment (thanhhungchu95) [inline config/routes.rb]: 1 space thôi

**Comment by tuquyet2709** [comment]
ok

**Pull request merged**